index.js

```diff
@@ -85,10 +102,16 @@ class PhotoViewManager {
       } else if (e.additionalEvent === 'pinchin') {
         this._transform(0, 0, 1);
       }
+      if(this.scale > 1){
```

Comment:
This code is duplicated.. can you put this into a separate method and use in both places ?